plugins/source/aws/client/spec/account.go

```diff
@@ -0,0 +1,21 @@
+package spec
+
+type Account struct {
+	ID              string   `json:"id" jsonschema:"required,minLength=1"`
+	AccountName     string   `json:"account_name,omitempty"`
+	LocalProfile    string   `json:"local_profile,omitempty"`
+	RoleARN         string   `json:"role_arn,omitempty" jsonschema:"pattern=^arn(:[^:\n]*){5}([:/].*)?$"`
+	RoleSessionName string   `json:"role_session_name,omitempty"`
+	ExternalID      string   `json:"external_id,omitempty"`
+	DefaultRegion   string   `json:"default_region,omitempty" jsonschema:"default=us-east-1"`
+	Regions         []string `json:"regions,omitempty" jsonschema:"minLength=1,default=us-east-1"`
```

Comment:
Why are we setting this value here? I think this will make this code innaccessible:
```
	localRegions := account.Regions
	if len(localRegions) == 0 {
		localRegions = awsPluginSpec.Regions
	}

```

Reply:
JSON schema validation doesn't perform setting of the defaults, it only documents them

Reply:
But the default value is not us-east-1 it is either `*` or empty

Reply:
Aha! Now that's a good thing to catch!

Reply:
removed `default` in c6a6bba